examples/angular/src/BUILD.bazel

```diff
@@ -268,8 +209,8 @@ pkg_web(
 ngsw_config(
     name = "pwa",
     src = ":prodapp",
+    out = "pwa-config",
```

Comment:
This just avoids a "rule and output have same name" (or something like that) warning